src/card/generate_card_assets.rb

```diff
@@ -25,7 +25,12 @@
 end
 
 catalogue_adapter = Card::CatalogueAdapters::JsonCatalogueAdapter.new("data/card_catalogue")
-generator = Card::Generators::VipsBasic.new
+
+card_templates = ->(card_spec) do
+  "assets/core/card/template/default.html"
+end
```

Comment:
I opted to go for a lambda here so that we could have arbitrarily complex template selection logic.

We'll have to see how this file balloons up and if we'll want to add in separate config files to pipe this information in.